src/App.js

```diff
@@ -120,6 +122,18 @@ const Guess = ({ guess, matches, word }) => {
   );
 };
 
+const welcome = ("Guess the five letter word");
+const newMatched3 = ["Are you clarvoyant!?","Time to buy some lotto tickets"];
+const newMatched2 = ["Outstanding!","Fantastic!"];
+const newMatched1 = ["Well done!","Impressive"];
+const newMisMatch2 = ["Good job!","Nice!"];
+const newMisMatch1 = ["You're making progress.","Keep chipping away at it."];
+const NoNew = ["Bummer.","A few options eliminated."];
+const winner = ["Congratulations! It took you ", " guesses."];
+const loser = ["You gave up after ", " guesses."];
+let newMatches = 0;
```

Comment:
We shouldn't be tracking this kind of state in variables outside of the App component. Instead, I think we should add a `useMemo` to track them, perhaps like:
`const latestMatchCounts = useMemo(() => { ...; return { matches, misMatches }; }, []);`

Reply:
tried this route for a bit, but ended up instead incorporating everything into `matches`.